content/influxdb3/enterprise/admin/license.md

```diff
@@ -0,0 +1,149 @@
+  {{< product-name >}} licenses authorize the use of the {{< product-name >}}
+  software. Learn how licenses work, how to activate and renew licenses, and more.
+menu:
+  influxdb3_enterprise:
+    name: Manage your license
+    parent: Administer InfluxDB
+weight: 101
+---
+
+{{< product-name >}} licenses authorize the use of the {{< product-name >}}
+software and apply to a single cluster. Licenses are primarily based on the
+number of CPUs InfluxDB can use, but there are other limitations depending on
+the license type. The following {{< product-name >}} license types are available:
+
+- **Trial**: 30-day trial license with full access to {{< product-name >}} capabilities.
+- **At-Home**: For at-home hobbyist use with limited access to {{< product-name >}} capabilities.
+- **Commercial**: Commercial license with full access to {{< product-name >}} capabilities.
+
+#### License feature comparison
+
+| Features       |           Trial           | At-Home |        Commercial         |
+| :------------- | :-----------------------: | :-----: | :-----------------------: |
+| CPU Core Limit |            256            |    2    |      _Per contract_       |
+| Expiration     |          30 days          | _Never_ |      _Per contract_       |
+| Multi-node     | {{% icon "check" "v2" %}} |         | {{% icon "check" "v2" %}} |
+| Commercial use | {{% icon "check" "v2" %}} |         | {{% icon "check" "v2" %}} |
+
+{{% caption %}}
+All other {{< product-name >}} features are available to all licenses.
+{{% /caption %}}
+
+## CPU limit
+
+Each {{< product-name >}} license limits the number of CPUs InfluxDB can use.
+The CPU limit is per cluster, not per machine. A cluster may consist of
+multiple nodes that share the available CPU limit.
+
+For example, you can purchase a 32-CPU Commercial license and set up an
+{{< product-name >}} cluster with the following:
+
+- 3 × writer nodes, each with 4 CPUs (12 total)
+- 1 × compactor node with 8 CPUs
+- 3 × query nodes, each with 4 CPUs (12 total)
+
+With the {{< product-name >}} Commercial license, CPU cores are purchased in
+batches of 8, 16, 32, 64, or 128 cores.
+
+### CPU accounting
+
+CPU cores are determined by whatever the operating system of the host machine
+reports as its core count. {{< product-name >}} does not differentiate between
+physical and virtual CPU cores.
+
+> [!Note]
+> If using Linux, InfluxDB uses whatever cgroup CPU accounting is active--for
+> example: `cpuset` or `cpu.shares`.
+
+## Activate a license
+
+Each {{< product-name >}} license must be activated, but the process of activating
+the license depends on the license type:
+
+- [Activate a Trial or At-Home license](#activate-a-trial-or-at-home-license)
+- [Activate a Commercial license](#activate-a-commercial-license)
+
+### Activate a Trial or At-Home license
+
+When starting the {{< product-name >}} server, it will ask you what type of
+license you would like to use. Select `trial` or `at-home` and provide your
```

Comment:
This should just be "\`home\`", not "\`at-home\`"

Reply:
```suggestion
license you would like to use. Select `trial` or `home` and provide your
```